packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,64 @@
 ---
-title: Upgrade guide
+title: Upgrade to Lit 3.x guide
 eleventyNavigation:
-  key: Upgrade guide
+  key: Upgrade to Lit 3.x guide
   parent: Releases
   order: 3
 versionLinks:
   v2: releases/upgrade/
 ---
 
-## Overview
+<div class="alert alert-info">
+
+If you are looking to migrate from Lit 1.x to Lit 2.x, see the [Lit 2.0 upgrade guide](/docs/v2/releases/upgrade/).
 
-Lit 3.0 has very few breaking changes from Lit 2.0. Any projects using Lit 2
-without deprecation warnings should not require any changes to upgrade to Lit 3.
+</div>
 
-Lit 3.0:
+## Overview
 
-- Drops support for IE11
-- Is published as ES2021
-- Removes a few deprecated APIs
+Lit 3.0 has very few breaking changes from Lit 2.0:
+
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated with the Lit 2.0 release have been removed.
+- SSR hydration support modules have moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
 
 Lit 3.0 should require no changes to upgrade from Lit 2.0 for the vast majority
 of users. Most apps and libraries should be able to extend their npm version
 ranges to include both 2.x and 3.x, like `"^2.7.0 || ^3.0.0"`.
 
-Lit 2.x and 3.0 are _interoperable_: templates, base classes, directives,
-decorators, etc., from one version of Lit will work with those from another.
+Lit 2.x and 3.0 are _interoperable_: templates, base classes, and directives from one version of Lit will work with those from another.
+
+## Decorator behavior has been unified
+
+To make the upgrade path from experimental decorators to standard decorators as
+smooth as possible, existing experimental decorators now support the standard
+spec, and work with the new `accessor` keyword. This is to make incremental
+migration to standard decorators possible.
+
+Once all decorator call sites in your project use the `accessor` keyword, you can
+build with `experimentalDecorators` set to `true` or `false` without a change in
+behavior.
+
+To make Lit decorators consistent between both decorator modes there have been some minor breaking
+changes to the experimental decorators:
+
+- `@property` and `@state` must be used on the setter for hand-written accessors. The getter no longer works.
+- `requestUpdate()` is called automatically for `@property` and `@state` decorated
+  accessors where previously that was the setters responsibility.
+- The value of an accessor is read on first render and used as the initial value for `changedProperties` and attribute reflection.
```

Comment:
Some things that I think are missing here:

- TypeScript 5.2 is required for standard decorators
- Babel v7.23.0 is supported with standard decorators

Due to the performance cost of standard decorators at the moment, and with no browsers shipping yet, we recommend most users stick with experimental decorators for the time being.

Reply:
This list is specifically calling out the breaking changes to existing experimental decorators. I'm not sure those dot points should go as part of this list. Maybe elsewhere in the upgrade guide, or in a dedicated decorator section.

List is introduced with `some minor breaking
changes to the experimental decorators:`

Reply:
Good point!